docs/dg/dev/internationalization-and-multi-store/add-new-store-in-multi-db-setup.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Adding new store in multi-db setup"
+description: Adding and deploying a new store in multi-db store setup requires additional steps and preparation. This guideline contains all the best practices you need to know.
+last_updated: Dec 02, 2024
+template: howto-guide-template
+originalLink: 
+originalArticleId: 
+redirect_from:
+---
+
+Setting up a new store in an existing multi-DB environment requires a well-prepared plan, to avoid disrupting the data and operations of existing stores. This guide provides a detailed procedure to release a new store within a region that already hosts other stores, ensuring a non-destructive deployment.
+
+## Initial planning and best practices
```

Comment:
Points look very valid for all types of multi store setups

Reply:
Agree, I know our existing documentation is incomplete - I tried to lookup multiple "multi store" articles we have - to link articles together, and it looks like a mess now.. but step by step we will improve it!